### PR TITLE
chore(deps): remove add and global and move @types/glob to devDeps

### PR DESCRIPTION
**Pull request by Zertz**
`add` and `global` are typically added by mistake when running `yarn add global` instead of `yarn global add`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,12 +25,10 @@
...
     "@actions/core": "^1.2.6",
     "@actions/github": "^4.0.0",
-    "@types/glob": "^7.1.3",
-    "add": "^2.0.6",
-    "glob": "^7.1.4",
-    "global": "^4.4.0"
+    "glob": "^7.1.4"
   },
   "devDependencies": {
+    "@types/glob": "^7.1.3",
```
Comment (ncipollo) [inline package.json:31]: I'm pretty sure this needs to stay in `dependencies`. If you clear out node_modules then perform a production only build typescript will fail to build (ex- `yarn clean && yarn install --production && yarn build`).

File: package.json [hunk 2]
```diff
@@ -25,12 +25,10 @@
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@actions/github": "^4.0.0",
-    "@types/glob": "^7.1.3",
-    "add": "^2.0.6",
```
Comment (ncipollo) [inline package.json:29]: Yup totally didn't mean to add these 🙃

**Review by ncipollo — CHANGES_REQUESTED** [review]
Thanks for the pull request! This is a good catch, I definitely didn't mean to include `add` and `global`. I think we just need to move `@types/glob` back to `dependencies` then this will be good to merge!

**Comment by Zertz** [comment]
Thanks for the feedback! I removed the `@types/glob` commit

**Review by ncipollo — APPROVED** [review]
Looks good 👍 
Thanks again!

**Pull request merged**